app/views/shopping_lists/index.html.erb

```diff
@@ -4,9 +4,21 @@
     <ul>
       <% @shopping_lists.reverse_each do |shopping_list| %>
         <li class="row">
-          <div class="columns small-10">
+          <div class="columns small-6">
             <%= render 'shopping_lists/editable_shopping_list_title', shopping_list: shopping_list %>
           </div>
+          <div class="columns small-4">
+            <% if shopping_list.primary %>
+              <i class="fi-check"></i>
+              Primary list
+            <% else %>
+              <%= link_to shopping_list_make_primary_path(shopping_list),
+                  method: :put,
```

Comment:
interesting.  I've never done that with a link.   Can you show me this on the page?

Reply:
Sure, these changes aren't in the deployed version, so just pop on over :)